Userland/Games/Hearts/Player.cpp

```diff
@@ -35,15 +35,15 @@ NonnullRefPtrVector<Card> Player::pick_cards_to_pass(PassingDirection)
     return cards;
 }
 
-Vector<CardWithIndex> Player::hand_sorted_by_fn(bool (*fn)(CardWithIndex&, CardWithIndex&)) const
+Vector<CardWithIndex> Player::hand_sorted_by_fn(Function<bool(CardWithIndex&, CardWithIndex&)> fn) const
 {
     Vector<CardWithIndex> sorted_hand;
     for (size_t i = 0; i < hand.size(); i++) {
         auto& card = hand[i];
         if (card)
             sorted_hand.empend(*card, i);
     }
-    quick_sort(sorted_hand, fn);
+    quick_sort<Vector<CardWithIndex>, Function<bool(CardWithIndex&, CardWithIndex&)> const&>(sorted_hand, fn);
```

Comment:
ew, please avoid this sort of mess, just move() in the damn function

Reply:
No, that's not possible. `dual_pivot_quick_sort` has two pivots, so it would call `move()` on the function twice.